app/src/main/java/com/github/se/polyfit/README.md

```diff
@@ -0,0 +1,24 @@
+## Documentation
+### Figma/material3 to Code
+
+In the [material3 doc](https://m3.material.io/components) you can find every usable component along with its "generic" appearance and functionality.  
+<br>For exemple: Navigation Bar
```

Comment:
I would instead say : For example for the Navigation Bar 
And delete the line bellow